testkit/src/main/scala/org/bitcoins/testkit/rpc/BitcoindRpcTestUtil.scala

```diff
@@ -852,7 +865,14 @@ trait BitcoindRpcTestUtil extends BitcoinSLogger {
       txid <- fundMemPoolTransaction(sender, address, amount)
       addr <- sender.getNewAddress
       blockHash <- sender.generateToAddress(1, addr).map(_.head)
-      _ <- hasSeenBlock(receiver, blockHash)
+      seenBlock <- hasSeenBlock(receiver, blockHash)
+      _ <- if (seenBlock) {
+        FutureUtil.unit
+      } else {
+        sender
+          .getBlockRaw(blockHash)
```

Comment:
How does this work compared to `getBlock()` which in turn is querying bitcoind`? Why will `getBlockRaw` succeed while `getBlock()` doesn't?

Reply:
`getBlock` returns information about the block (a `GetBlockResult`) vs `getBlockRaw` returns an actual `Block`

Reply:
Why would `getBlockRaw` work and `getBlock` wouldn't work in the same bitcoind instance

Reply:
The problem wasn't that they were failing, the problem was that the `receiver` had not seen the block, so we use `getBlockRaw` on the `sender` and give it to the `receiver`